tests/python/pants_test/util/test_socket.py

```diff
@@ -44,9 +49,18 @@ def test_recv_max_larger_than_buf(self):
     self.server_sock.sendall(b'A' * double_chunk)
     self.assertEqual(self.buf_sock.recv(double_chunk), b'A' * double_chunk)
 
-  @mock.patch('pants.util.socket.select.select', **PATCH_OPTS)
-  def test_recv_check_calls(self, mock_select):
-    mock_select.return_value = ([1], [], [])
+  @mock.patch('selectors.DefaultSelector' if PY3 else 'select.select', **PATCH_OPTS)
+  def test_recv_check_calls(self, mock_selector):
```

Comment:
thanks for adding the test!

Reply:
To clarify, I didn't add any new tests—only modified the original to work with `selectors` instead of `select`.